table/table_test.cpp

```diff
@@ -155,7 +157,17 @@ int main() {
     assert(table.filter(columns1, comparisons1)[0].entries(0).num() == 1);
     
     table.edit_rows(columns3, comparisons3, columns3, comparisons4);
-    assert(table.filter(columns1, comparisons1).size() == 3);
+    assert(table.filter(columns3, comparisons4).size() == 4);
     
     cout << "Passed Update Tests!" << endl;
+    /* Passed Update Tests */
+
+    /* Deletion Tests */
+    cout << "Starting Deletion Tests..." << endl;
```

Comment:
When writing tests, please separate tested codepaths in their own helper functions. This will help with organizing the overall main function so its not one large function.